source/2_platform/rex_directx/src/directx_context.cpp

```diff
@@ -0,0 +1,21 @@
+#include "rex_directx/log.h"
+#include "rex_renderer_core/context.h"
+
+// NOLINTBEGIN(modernize-use-nullptr,-warnings-as-errors)
+
+namespace rex
+{
+  namespace context
+  {
+    namespace backend
+    {
+      //-------------------------------------------------------------------------
+      bool create(void* /*unused*/)
+      {
+        return true;
```

Comment:
I assume this will be implemented in the future?

Reply:
Yes it will, once this PR is in main